lambdas/service/app.py

```diff
@@ -1987,3 +1954,41 @@ def gs_url(file_uuid, version):
 
 class GatewayTimeoutError(ChaliceViewError):
     STATUS_CODE = 504
+
+
+@app.route('/integrations/add_portal', methods=['POST'])
```

Comment:
I had missed these two endpoints in earlier reviews. We need to disable or remove them. The easiest way to disable would be to embed the method def in `if False`. Instead of `False` an environment variable could be used, with a wrapper property in `azul.config` etc, but that seems overkill.

Reply:
Is there any reason to keep them in the source code at all? I only added them for demo purposes, and with the script added in #1511 they aren't even needed for that anymore. I was expecting that the commit that added them would be omitted when rebase/squashing once we were ready to merge.

Reply:
> Is there any reason to keep them in the source code at all?

I was undecided but it sounds like you're leaning towards removing them. Please do so, ideally by removing the commit that added them.

>  was expecting that the commit that added them would be omitted when rebase/squashing once we were ready to merge.

If you want the reviewer to remove a commit during squashing, you can add the prefix DELETE to the commit title as documented in the contributing guidelines.

Reply:
Noted. The commit has been removed.